types/chai/chai-register-should-tests.ts

```diff
@@ -0,0 +1,2610 @@
+/// <reference types="node" />
+import { assert, config, expect, Should, use, util } from "chai";
+import 'chai/register-should';
```

Comment:
I'm not clear on how DefinitelyTyped tooling works: I would have expected this `import` to pollute the types for the other tests and cause them to fail, but they don't. Is that expected?